code/ai/ai_profiles.cpp

```diff
@@ -472,6 +472,18 @@ void parse_ai_profiles_tbl(const char *filename)
 
 				set_flag(profile, "$use actual primary range:", AI::Profile_Flags::Use_actual_primary_range);
 
+				if (optional_string("$constant radii for subsystem path points:")) {
+					int path_radii;
+					stuff_int(&path_radii);
+					if (path_radii > 0) {
+						profile->subsystem_path_radii = path_radii;
+					} else {
+						mprintf(("Warning: \"$constant radii for subsystem path points:\" should be greater than 0 (read %i). Value will not be used. ", path_radii));
+					}
+				}
+
+				set_flag(profile, "$use model path point radii for subsystem path navigation:", AI::Profile_Flags::Use_subsystem_path_point_radii);
```

Comment:
I would suggest being consistent with the option names here: you have "subsystem path points" vs. "subsystem path navigation".

Also "constant radii" is IMO not very intuitive.  And "radii" is technically correct but it's not clear (to the lazy reader) why you're using the plural.

I suggest "override radius for model path points" for the first option and "use POF radius for model path points" for the second option.  (And they should be documented on the wiki page.)

The options *are* applicable for both docking paths and subsystem paths, right?

Reply:
ATM these options are only applicable for use in the function `ai_big_maybe_follow_subsys_path` which is only called to attack subsystems (as mentioned in the initial post of this comment :) ). Docking did not have that problem of ships not being able to follow/complete paths because the AI slows down considerably when docking and damp values seem not to be used (similar to bay paths). 

Good suggestions for the name changes.

Reply:
Yeah, my suggestion of "model path points" was with the assumption that they would be used for all paths.  So "subsystem path points" is fine.